0000-template-process.md

```diff
@@ -0,0 +1,99 @@
+---
+process: (fill me in with a unique ident, my_new_process)
+start-date: (fill me in with today's date, YYYY-MM-DD)
+author: (name of the main author)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+modifies:
+ - [RFC 0000 my_old_process]()
+category: process
+---
+
+<!--
+If you are proposing to change a process with regard of how the nix community
+conducts, then use this template. Some examples are, without being an exhaustive list:
+
+- Change the RFC process, the organization of the issue tracker or the forum
+- Change community workflows or other comunity infrastructure
+- Amend the code of conduct and similar high level normative documents
+-->
+
+# Summary
+[summary]: #summary
+
+<!-- One paragraph to resume this document (motivation and future process). -->
+
+In order to address "X", I/we propose to "Y".
+
+# Classification
+[classification]: #classification
+
+<!-- Please check the relevant boxes (typically one) -- or add your own. -->
+
+- [ ] general habits and decision making
+- [ ] core technical protocols & processes
+- [ ] contributer efficiency support
+
+# Motivation
+[motivation]: #motivation
+
+<!-- What's wrong? Please feel encouraged to benchmark us against other
+(open source or other) ecosystems. -->
+
+# Current Process
+[as-is]: #current-process
+
+<!-- Describe the current process as it is observed out in the wild.
+If there has been a previous RFC for this process, please mention it,
+but prefer the state of the world "as-is". In a final paragraph, please
+describe its shortcomings and how they relate to the motivation.
+
+Make use of BPMN 2.0 notation, if you'd find that useful. -->
+
+# Future Process
+[to-be]: #future-process
+
+<!-- Describe the future process how you imagine it to be.
+In a final paragraph, please describe how this would satisfy the motivation.
+Please be explicit, if it only party addresses the motivation.
+
+Make use of BPMN 2.0 notation, if you'd find that useful. -->
+
+# Roles & Stakeholders
+
+<!-- 
+Stakeholders are people who have an interest in the outcome of this RFC.
+
+Please describe in abstract terms the roles involved in this process
+and how they are affected by this process change. Plotting estimated
+/ abstract time requirements of _as-is_ against _to-be_ is a plus.
+The idea is to get a better sense of the stakeholders of this process
+and their respective interestes and estimate the associated total
+costs imposed (mostly in time, can be negative) to the community.
```

Comment:
I do appreciate the fact that having a separate "process" category allows us to add guidelines to this document, thereby making it clearer what is expected.

I am not sure that I would want to go through this kind of jargon to want to do that though. I'd remove mentions of BPMN, plotting time requirements and even of the word "stakeholders". The corporate-sounding language rubs me the wrong way, personally.

Reply:
I think usage of language should be both neutral and concise. We concluded in the shepherd meeting, that stakeholder is the most concise english word available here, even though originating from a organizational-philosophical context.

I think time burdon have repetedly come up as a counter argument in many process amendements, especially time requirements on maintainers.

In my opinion, suggesting to explicitly treat maintainer's interests and conscientiously assess time requirements that any change places on them might help to promote better informed discussions.